app/tests/unit/components/Layout/__snapshots__/Header.test.tsx.snap

```diff
@@ -2,48 +2,48 @@
 
 exports[`matches the last accepted Snapshot 1`] = `
 <header
-  className="jsx-2785271651 "
+  className="jsx-2264600173 "
```

Comment:
We're trying to improve our unit testing to have more functional assertions, as compared to mainly snapshots.

Since this work is turning the Header into more of a container that interacts with data (the user object), it might be a good time to add some functional assertions based on whether or not the user is logged in.